content/docs/learning-guide/tutorials/environments/generate-terrain.md

```diff
@@ -0,0 +1,303 @@
+
+* The **Dimensions** property of the **Axis Aligned Box Shape** component specifies the terrain area. The **Dimensions: Z** property, in particular, specifies the height of the terrain. Its value should not exceed the range defined in the **Terrain World** level component by the **Min Height** and **Max Height** properties.
+
+* The **Translate** property of the **Transform** component of the Terrain Spawner entity places the terrain entity in the level. The translate position of the Terrain Spawner entity is at the center of the box shape that defines the terrain area. This means that if the box shape is a 512 unit cube, and the **Translate** property is `0.0` on each axis, then the terrain extends 256 units in each direction from the origin of the level. Care should be taken to ensure that the **Translate: Z** value places the box shape within the range defined by the **Min Height** and **Max Height** properties of the **Terrain World** level component.
+
+## Apply terrain materials
+
+There are two types of materials that are needed for terrain, the *macro material* and *detail materials*.
+
+The macro material is a basic material that applies to the entire terrain. It supports a simple color texture and a normal texture. The macro material is displayed on terrain that is far from the camera.
+
+Detail materials are standard materials that you can create with [Material Editor](/docs/atom-guide/look-dev/materials/material-editor/). They are assigned to the terrain through surface tag names that are generated by gradients, similar to how this tutorial used an image gradient to create elevation. This allows you to assign many materials across a large terrain surface and blend between them. Detail materials are displayed on terrain surfaces that are close to the camera.
+
+{{< tip >}}
+In the **Terrain World Renderer** level component, there is a property group named **Detail material configuration** that you can use to configure how far from the camera to switch from detail material to macro material, and the distance over which to blend between the materials.
+{{< /tip >}}
+
+The following example image demonstrates various material assignments and how they interact on a terrain surface:
+
+{{< image-width src="/images/learning-guide/tutorials/environments/terrain-materials-example.png" width="800" alt="An example of macro and detail materials applied to a terrain." >}}
+
+In this example, there are four materials:
+
+* Magenta - This is th macro material. It's displayed on terrain that is 512 units from the camera. The blend distance between this macro material and the detail materials is 54 units.
```

Comment:
"th" -> "the"
"512 units" -> "512 meters"
"54 units" -> "64 meters"

Reply:
Fixed